src/components/NewCommentForm.tsx

```diff
@@ -1,8 +1,121 @@
-import React from 'react';
+import classnames from 'classnames';
+import React, { useState } from 'react';
+import { CommentData } from '../types/Comment';
+
+type Props = {
```

Comment:
Good job on defining the `Props` type for your component. This helps to ensure type safety for the props that your component receives.